README.md

```diff
@@ -72,6 +72,8 @@ First, you need to fork the `react-play` repo. You can do this by clicking the `
 
 Once forked, you can clone the repo by clicking the `Clone or Download` button on the top right corner of forked repo.
 
+Please change the directory after cloning the repository using the ```cd <folder-name>``` command.
```

Comment:
Don't use three backticks. one is enough.

Reply:
Ok